website/routes.py

```diff
@@ -1151,6 +1151,9 @@ def make_url_map(app):
         Rule([
             '/project/<pid>/draft/<draft_id>/',
         ], 'delete', project_views.drafts.delete_draft_registration, json_renderer),
+        Rule([
+            '/project/<pid>/draft/<draft_id>/submit/',
```

Comment:
Standard confirmation question: have these new routes been approved by the routing Tsar?